haystack/nodes/answer_generator/openai.py

```diff
@@ -101,14 +101,14 @@ def predict(self, query: str, documents: List[Document], top_k: Optional[int] =
 
         Example:
          ```python
-            |{
-            |    'query': 'Who is the father of Arya Stark?',
-            |    'answers':[Answer(
-            |                 'answer': 'Eddard,',
-            |                 'score': None,
-            |                 ),...
-            |              ]
-            |}
+            {
+                'query': 'Who is the father of Arya Stark?',
+                'answers':[Answer(
+                             'answer': 'Eddard,',
+                             'score': None,
+                             ),...
+                          ]
+            }
```

Comment:
This is probably why the pipes were removed in the docs above. The pipes are useful, please keep them (see #3006 too).

Reply:
Sounds good I'll revert the pipe changes.